schemas/Size.yaml

```diff
@@ -0,0 +1,13 @@
+required:
+ - name
+ - value
+ - unit
+properties:
+  name:
+    type: string
+  value:
+    type: string
+  unit:
```

Comment:
Appreciate having a first class object to represent these (CoA) measurements as it IS definitely required in our space / scope. I would push back on Size being a useful name for measurements that usually detail weight and percentage. I get that "serving size" (implemented above) is a colloquial and is perhaps where this abstraction came from? I would suggest that an "Amount" is a multi-unit label that is inclusive of weight and percentage, and further that actually the popular term serving size SHOULD be serving amount. Sorry if this seems pedantic, I don't feel that it is.

Reply:
ok.